lfucg-exactions/server/plats/viewsets.py

```diff
@@ -7,14 +7,59 @@ class SubdivisionViewSet(viewsets.ModelViewSet):
     serializer_class = SubdivisionSerializer
     queryset = Subdivision.objects.all()
 
+    def get_queryset(self):
+        queryset = Subdivision.objects.order_by('name')
+
+        query_text = self.request.query_params.get('query', None)
+        if query_text is not None:
+            query_text = query_text.lower()
+            tmp_set = queryset.filter(name__icontains=query_text)
```

Comment:
I think it might be better if users can search for part of a subdivision name (sometimes people only remember a portion of a name) for simplest search.